packages/block-library/src/archives/style.scss

```diff
@@ -0,0 +1,3 @@
+.wp-block-archives-dropdown {
+	margin-bottom: 1.5em;
```

Comment:
@samikeijonen provided the value to me as a suggestion.

Reply:
It would be good to have a variable for this. But given this is front-end stuff, and we mean to provide some basics, and in this case I think it's okay to use the em unit, as many of the basic elements are born with browser vanilla ems. 

In this case, though, I'd take a cue from the `figure` element, though, and make it 1em:

![screenshot 2018-11-20 at 19 54 24](https://user-images.githubusercontent.com/1204802/48795981-b2c67900-ecfe-11e8-9a85-b57564c76a47.png)

Reply:
@jasmussen thanks

Reply:
In some of the existing block margin bottom in the front-end is `1.5em`, that's why I suggested that for consistency.